micro-rdk-installer/src/error.rs

```diff
@@ -49,6 +49,8 @@ pub enum Error {
     SerialConfigError(String),
     #[error("No command received")]
     NoCommandError,
+    #[error("Uncategorized Error: {0}")]
+    Uncategorized(String),
```

Comment:
Generally it's better to use Box<dyn Error + Sync ....> to represent errors from other libraries then you can use #transparent 
I see you use it on Options in this case you should either return the same error as other part of the code or wrap it in a function  returning an error.